addons/hr/views/hr_plan_views.xml

```diff
@@ -180,5 +180,33 @@
                 </form>
             </field>
         </record>
+
+        <record id="mail_activity_type_form_inherit" model="ir.ui.view">
```

Comment:
This will hide the fields for every app, not only HR
I'm not sure that's what we want :thinking: